src/index.md

```diff
@@ -8,53 +8,73 @@ LegalTech.
 The Exonum framework is oriented towards creating permissioned blockchains,
 that is, blockchains with the known set of blockchain infrastructure providers.
 
-Exonum uses [the Rust programming language][rust] to achieve utmost execution
-safety;
+Exonum uses [the Rust programming language][rust] in its core to achieve utmost
+execution safety;
 [service-oriented architecture][wiki:soa] to provide extensibility, flexibility,
 and modularity; and client-side verification based on
 [cryptographic commitments][wiki:commitment]
 (Merkle and Merkle Patricia trees) to ensure transparency of the system
 and client security.
 
+Exonum provides [the Java Binding tool][ejb] together with a software
```

Comment:
[ejb] — at the moment it links to the repository — shan't we link to the Java Binding page instead (which links to repo in ["How to Run a Service"](https://exonum.com/doc/version/latest/get-started/java-binding/#how-to-run-a-service) section), because the repository just links back to the documentation and to the cryptocurrency demo (see https://github.com/exonum/exonum-java-binding/tree/master/exonum-java-binding)?